src/main/java/com/walking/intensive/chapter2/task6/Task6.java

```diff
@@ -31,8 +39,20 @@ static int getLcm(int m, int n) {
      * <p>Если входные данные некорректны - метод должен возвращать -1.
      */
     static int getGcd(int m, int n) {
-        // Ваш код
-        return 0;
+        if (m <= 0 || n <=0) {
```

Comment:
Error: eckstyle] [ERROR] /home/runner/work/intensive-tasks-20[24](https://github.com/KFalcon2022/intensive-tasks-2024/actions/runs/12032116316/job/33578643146?pr=400#step:4:25)/intensive-tasks-2024/src/main/java/com/walking/intensive/chapter2/task6/Task6.java:42:[25](https://github.com/KFalcon2022/intensive-tasks-2024/actions/runs/12032116316/job/33578643146?pr=400#step:4:26): '<=' is not followed by whitespace. [WhitespaceAround]